modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
```

Comment:
I think the port only makes sense when talking about the direction of the request as well. For example, for `Scanner and Scanner-DB | TCP | 5432`, this only makes sense when it's clear that Scanner reaches out to Scanner DB at port 5432.

At the moment, the table makes it seem like the communication may go either way and use that same port for both directions.